app/prism/PrismLogic.scala

```diff
@@ -99,4 +101,26 @@ object PrismLogic {
       }
     }.getOrElse(true)
   }
+
+  /**
+    * @return Percentiles of instance AMIs age
+    */
+  def instancesAmisAgePercentiles(amis: List[AMI]): Percentiles = {
+    val ages = amis.flatMap { ami =>
+      val amiAge = ami.creationDate.map(DateUtils.daysAgo).getOrElse(0)
```

Comment:
I think a 'fairer' implementation would be to ignore ami's lacking a creation date, rather than assume they are brand-new. The former leaves the stats unchanged, this implementation moves us towards our goal without any evidence we've made progress. Better to flatten this, I think.